nginx_proxy/DOCS.md

```diff
@@ -82,6 +83,10 @@ The filename(s) of the NGINX configuration for the additional servers, found in
 If enabled, configure Nginx with a list of IP addresses directly from Cloudflare that will be used for `set_real_ip_from` directive Nginx config.
 This is so the `ip_ban_enabled` feature can be used and work correctly in /config/customize.yaml.
 
+### Option `show_version` (optional)
+
+If disabled, configure Nginx to hide its version in response Headers and error pages.
+
```

Comment:
We don't have to make this configurable IMHO. Just hide it always.

Reply:
In that case, I will rebase this PR and convert it to completely hiding it. I didn't want to make such a change without discussing it. Give me 5m.

Reply:
You also need to bump the version (3.2.0) and add an entry to the changelog file